deployer/src/lib.rs

```diff
@@ -104,6 +104,10 @@ pub struct StartOptions {
     #[structopt(long)]
     pub no_rest: bool,
 
+    /// Disable the CSI Controller
+    #[structopt(long)]
+    pub no_csi: bool,
```

Comment:
We should reverse it, so that we don't launch CSI by default, since most of the existing uses of the deployer (cargo tests and local debugging) don't use the CSI.

Reply:
Good point.